### PR TITLE
Debian

### PR DESCRIPTION
**Pull request by fepitre**

### CODE REVIEW

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Maintainer: Frédéric Pierret (fepitre) <frederic.pierret@qubes-os.org>
+Build-Depends:
+ debhelper (>= 10~),
+ inkscape,
+ imagemagick,
+ librsvg2-bin,
+ netpbm,
+ fonts-roboto,
+ qubes-utils,
+ python3-cairo
```
Comment (marmarek) [inline debian/control]: `python3-cairo` should be rather in qubes-utils. See b71235cc

File: qubes-artwork.spec.in [hunk 2]
```diff
@@ -55,27 +50,33 @@ Summary:    Qubes OS theme for EFI boot manager (rEFInd)
 %description efi
 Qubes OS theme for rEFInd boot manager
 
+%package anaconda
```
Comment (marmarek) [inline qubes-artwork.spec.in:53]: What will pull in this new package?

**Review by marmarek — CHANGES_REQUESTED** [review]

**Comment by marmarek** [comment]
Also, the Debian bullseye build fails - failed to install build deps.

File: qubes-artwork.spec.in (hunk 2 above)
Comment (fepitre) [reply qubes-artwork.spec.in:53]: This https://github.com/QubesOS/qubes-lorax-templates/pull/7 and https://github.com/QubesOS/qubes-qubes-release/pull/9. Forgot to PR them. A submodule update in installer is thus needed.

File: debian/control (hunk 1 above)
Comment (fepitre) [reply debian/control]: https://github.com/QubesOS/qubes-linux-utils/pull/58

**Review by marmarek — APPROVED** [review]

**Pull request merged**